stable/kong/README.md

```diff
@@ -69,9 +69,9 @@ and their default values.
 | env                               | Additional [Kong configurations](https://getkong.org/docs/latest/configuration/)               |
 | runMigrations                     | Run Kong migrations job                                                | `true`                |
 | readinessProbe                    | Kong readiness probe                                                   |                       |
-| readinessProbe.httpGet.schema     | Schema used by readyness probe. Set to HTTP if admin.useTLS = false    | `HTTPS`               |
+| readinessProbe.httpGet.scheme     | Scheme used by readyness probe. Set to HTTP if admin.useTLS = false    | `HTTPS`               |
 | livenessProbe                     | Kong liveness probe                                                    |                       |
-| livenessProbe.httpGet.schema      | Schema used by liveness probe. Set to HTTP if admin.useTLS = false     | `HTTPS`               |
+| livenessProbe.httpGet.scheme      | Scheme used by liveness probe. Set to HTTP if admin.useTLS = false     | `HTTPS`               |
```

Comment:
@marcostvz  personally I don't think we need to add this here, user can also set different test probe, like tcp probe or command probe. Warning on line 17 should be enough.

Reply:
Good enough for me :) Removing it